src/MeasureUtil/MeasureUtil.ts

```diff
@@ -27,14 +27,15 @@ class MeasureUtil {
    * @return {number} The length of line in meters.
    */
   static getLength(line: OlGeomLineString, map: OlMap, geodesic: boolean = true, radius: number = 6371008.8): number {
+    const decimalPrecision = Math.pow(10, 6);
```

Comment:
Thanks for your contribution @Batiste-Yole-SP 

In my opinion it would make sense to make precision configurable via a function parameter having `Math.pow(10, 6)` as default. Could you please change this accordingly?

Reply:
You are welcome,
I change it to set the precision as a new getLength parameter.
I tried to fit with the formatLength method to implement it.
So the parameter is waiting for a short integer corresponding to the decimal precision awaited.

Reply:
Looks good :+1:

Please reformat your commit messages according to the https://www.conventionalcommits.org/en/v1.0.0/ specification (e.g. `fix: … ` with `BREAKING CHANGE` commit footer) such that semantic release plugin will release a proper version.

Reply:
Sorry I'm not used with rebase but I manage to set a BREAKING CHANGE footer on the commits.
It should be OK.

Reply:
Can be merged now.

Reply:
Hello, a quick bump on this one, do you have a visibility of merge so that we can use it in react-geo lib ?